src/modules/admin/components/formRules/FormRuleTable.tsx

```diff
@@ -127,75 +46,39 @@ const FormRuleTable: React.FC<Props> = ({
     getVariables: (values) => ({
       input: { input: values as FormRuleInput, id: selectedRule?.id || '' },
     }),
-    onCompleted: () => {},
+    onCompleted: () => {
+      evictQuery('formDefinition', { id: formId });
```

Comment:
Maybe could use more precise cache manipulation to _just_ invalidate/refetch the `formDefinition`'s `projectMatches`, and not the entire record.

Reply:
Since this is related to editing, I'd like to address it in https://github.com/open-path/Green-River/issues/6109 which I'll pick up next!